<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>1062</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>969</width>
     <height>1006</height>
    </size>
   </property>
   <widget class="QLabel" name="AltitudeTag">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Altitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedTag">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="AltitudeVal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedVal">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Voltage1Label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>339</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>379</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="AccelerationTag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>419</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceleration</string>
    </property>
   </widget>
   <widget class="QLabel" name="AccelerationVal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>419</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PressureVal">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedTag_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pressure</string>
    </property>
   </widget>
   <widget class="QLabel" name="MissionTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>219</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mission Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="MissionTimeVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>219</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PacketCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>239</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Packet Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="PacketCountVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>239</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Latitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="LatitudeVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>259</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Longitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="LongitudeVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>279</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HeadingZVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>299</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="Map" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>941</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TelemetryText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>941</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cansatLOGO">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Team ViSat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VIT University, Vellore</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>INDIA</string>
    </property>
   </widget>
   <widget class="QLabel" name="Temperature">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="TemperatureVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>AltitudeTag</zorder>
   <zorder>SpeedTag</zorder>
   <zorder>Voltage1Label</zorder>
   <zorder>label_2</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>AccelerationTag</zorder>
   <zorder>AccelerationVal</zorder>
   <zorder>widget_3</zorder>
   <zorder>SpeedTag_2</zorder>
   <zorder>SpeedVal</zorder>
   <zorder>PressureVal</zorder>
   <zorder>AltitudeVal</zorder>
   <zorder>MissionTime</zorder>
   <zorder>MissionTimeVal</zorder>
   <zorder>PacketCount</zorder>
   <zorder>PacketCountVal</zorder>
   <zorder>Latitude</zorder>
   <zorder>LatitudeVal</zorder>
   <zorder>Longitude</zorder>
   <zorder>LongitudeVal</zorder>
   <zorder>HeadingZVal</zorder>
   <zorder>Logo</zorder>
   <zorder>Map</zorder>
   <zorder>TelemetryText</zorder>
   <zorder>cansatLOGO</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>Temperature</zorder>
   <zorder>TemperatureVal</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
